templates/windows-10/default-browser.html

```diff
@@ -0,0 +1,128 @@
+<!-- Any instances of {{ snippet_id }} are replaced with the ID number for the
+     snippet using this template. -->
+
+<div class="snippet" id="{{ snippet_id }}-fx-default">
+    <div id="{{ snippet_id }}-content"></div>
+
+    <script type="text/javascript">
+      //<![CDATA[
+
+      (function() {
+        var snippet = document.getElementById('{{ snippet_id }}-fx-default');
+        snippet.addEventListener('show_snippet', function () {
+          var buttons = snippet.getElementsByClassName('launch-default');
+          var snippet_content = document.getElementById('{{ snippet_id }}-content');
+
+          isDefault('appinfo', function yesDefault () {
+            sendMetric('impression-is-default');
+            snippet_content.innerHTML = '<a href="#" class="launch-default"><img class="icon" src="{{ icon_default }}" /></a><p>{{ text_default|escape }}</p>';
+
+            for (var i=0; i < buttons.length; i++) {
+              buttons[i].addEventListener('click', function (e) {
+                e.preventDefault();
+                sendMetric('conversion-is-default-click', function () {
+                  window.location = '{{ url_default|safe }}';
+                });
+              }, false);
+            }
+          }, function noDefault () {
+            sendMetric('impression-is-not-default');
+            snippet_content.innerHTML = '<a href="#" class="launch-default"><img class="icon" src="{{ icon_default }}" /></a><p>{{ text_not_default|escape }}</p>';
+
+            for (var i=0; i < buttons.length; i++) {
+              buttons[i].addEventListener('click', function (e) {
+                e.preventDefault();
+                Mozilla.UITour.setConfiguration('defaultBrowser');
+                sendMetric('conversion-is-not-default-click');
```

Comment:
Tried the `setDefault` default functionality on linux and it works. On the UX front user does not get any message that this succeeded though, maybe we still want to redirect them to `url_default`?